shared/middleware/assets_api.rb

```diff
@@ -35,7 +35,16 @@ class AssetsApi < Sinatra::Base
     s3 = Aws::S3::Client.new(region: 'us-east-1')
     s3.list_objects(bucket:CDO.assets_s3_bucket, prefix:prefix).contents.map do |fileinfo|
       filename = %r{#{prefix}/(.+)$}.match(fileinfo.key)[1]
-      {filename:filename, size:fileinfo.size}
+      mime_type = Sinatra::Base.mime_type(filename.split('.').last)
+      case mime_type.split('/').first
+        when 'image'
+          category = 'image'
+        when 'audio'
+          category = 'sound'
```

Comment:
Probably fine to keep this as 'audio' for consistency.  Then you can write it without the `case` statement, e.g.:

```ruby
category = mime_type.split('/').first
halt(415, "Unexpected mime type: #{mime_type}") unless %w(image audio).include? category
```